### PR TITLE
Issue #75

### PR DESCRIPTION
**Pull request by AlexPernot**
Expanded the size of the arrow divs, and reajusted text alignment.
This way, the active zones of the arrows are bigger.

-- User @romainchalut also helped me.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -23,6 +23,8 @@ html, body {
     position: absolute;
     bottom: 0;
     z-index: 5;
+	width: 20%;
```
Comment (Yoric) [inline css/style.css]: Please fix the indentation.

**Comment by Yoric** [comment]
This looks almost good. Can you fix the indentation before I merge your pull request?

**Comment by AlexPernot** [comment]
Done!

**Comment by Yoric** [comment]
I'd prefer a single commit. Can you fold them using `git rebase -i`?

**Comment by AlexPernot** [comment]
Should be good now.

**Comment by Yoric** [comment]
Looks good, thank you very much.

**Pull request merged**